pkg/util/azureclient/mgmt/insights/activitylogs.go

```diff
@@ -4,25 +4,38 @@ package insights
 // Licensed under the Apache License 2.0.
 
 import (
-	mgmtinsights "github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights"
 	"github.com/Azure/go-autorest/autorest"
 	"github.com/Azure/go-autorest/autorest/azure"
+
+	//"github.com/drewandersonnz/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights"
```

Comment:
```suggestion
	//"github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights"
```

Reply:
You're right, that was a mistake due to trying to finish it quickly... :-( 
Good spotting! That location doesn't exist anymore.